src/main/scala/org/constellation/util/Periodic.scala

```diff
@@ -0,0 +1,35 @@
+package org.constellation.util
+
+import java.util.concurrent.{ScheduledThreadPoolExecutor, TimeUnit}
+
+import scala.concurrent.Future
+import scala.util.Try
+
+abstract class Periodic(threadName: String, periodSeconds: Int = 1) {
```

Comment:
Long-term I'm not sure if I prefer this thread per class approach to some kind of scheduler object, but it works fine for now.

Glad we're less reliant on akka.

Reply:
We can just make the scheduler an argument to this and have a pool for periodic stuff.